panel/src/components/Layout/ItemImage.vue

```diff
@@ -18,7 +18,6 @@
     <k-aspect-ratio v-else :ratio="ratio">
       <k-icon
         :color="$helper.color(image.color)"
-        :size="size"
```

Comment:
Not supporting `size` here doesn't seem right - the component should keep supporting it.

Reply:
You are right. That was obviously just the most lazy way to solve it :)

Reply:
Oh wait, the icon actually never really supported the size attribute. It's just a computed prop that handles the size according to the layout prop.

Reply:
Although this means that also emojis etc. will be sized differently now - not just your new file icons. Not sure if that's what we want.

Reply:
I'll give it another look tomorrow. But I think it's fine if the emojis are also just the regular size in cards. If someone complains we can still turn it back :)

Reply:
Looks good to me …
![Screenshot 2022-06-14 at 10 41 14](https://user-images.githubusercontent.com/24532/173534070-3b07d504-084d-45da-b2e4-26d915e42d53.png)

Reply:
I’m going to be the one that says it was better before 😬 It helps distinguish pages better I think.

![Capture d’écran 2022-07-14 à 14 46 25](https://user-images.githubusercontent.com/793344/178985702-1c4ee53d-c788-4caa-a9da-266b99938739.png)

![Capture d’écran 2022-07-14 à 14 46 16](https://user-images.githubusercontent.com/793344/178985714-641716b3-3226-44dc-8672-24033ba176ef.png)